genai-function-calling/spring-ai/env.example

```diff
@@ -1,8 +1,10 @@
 # Update this with your real OpenAI API key
 OPENAI_API_KEY=
 
-# Uncomment to use Ollama instead of OpenAI
-# OPENAI_BASE_URL=http://localhost:11434/v1
+# Uncomment to use Ollama instead of OpenAI.
+# Note: OpenAI's OPENAI_BASE_URL var ends in /v1, while this doesn't, until:
```

Comment:
BTW, just a reminder that spring's variable is `SPRING_AI_OPENAI_BASE_URL` so maybe it's not as weird they behave differently, though indeed BASE_URL is shared